specification/amqp_0.9.1/src/main/scripts/org/kaazing/specification/amqp/ws/open/identity/response.rpt

```diff
@@ -26,7 +26,7 @@ read header "Upgrade" /(?i:websocket)/
 read header "Connection" /(?i:Upgrade)/
 read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
 read header "Sec-WebSocket-Version" "13"
-read header "Authorization" "Basic am9lOndlbGNvbWU="
+read header "Authorization" ${authHeader} #authHeader defined in request.rpt
```

Comment:
You'll need to define the properties in this script as well so it can be used alone (without the request.rpt)